client/src/modules/Score/data/getColumns.tsx

```diff
@@ -80,15 +30,15 @@ export function getColumns(props: Props): ColumnTypeWithName<StudentScoreWithCro
       dataIndex: 'rank',
       key: 'rank',
       width: 50,
-      sorter: sorter('rank'),
+      sorter: 'rank',
```

Comment:
So I suppose it the same as `sorter: true`

Reply:
apparently not, I thought of it too, but when I set it true sorter doesn't work properly

Reply:
I don't get why sorter works in the first place with `string`, and works properly

Reply:
Got you